### PR TITLE
docs(cce): Correct the descriptions of the CCE documents

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
CCE documents currently have many problems such as non-compliance with specifications, incorrect parameter names, and incorrect parameter descriptions.

**Which issue this PR fixes**:
```
NONE
```

**Special notes for your reviewer**:

**Release note**:

```release-note
1. correct the wrong parameter names, types and descriptions.
2. correct the format of noun abbreviations.
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: docs/data-sources/cce_cluster.md [hunk 1]
```diff
@@ -22,22 +22,24 @@ data "huaweicloud_cce_cluster" "cluster" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String) The region in which to obtain the cce clusters. If omitted, the provider-level region
-  will be used.
+* `region` - (Optional, String) Specifies the region in which to obtain the CCE cluster. If omitted, the provider-level
+  region will be used.
 
-* `name` - (Optional, String)The Name of the cluster resource.
+* `name` - (Optional, String) Specifies the Name of the cluster.
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cce_cluster.md]: s/Name/name

File: docs/data-sources/cce_node.md [hunk 2]
```diff
@@ -22,30 +22,32 @@ data "huaweicloud_cce_node" "node" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String) The region in which to obtain the cce nodes. If omitted, the provider-level region will
+* `region` - (Optional, String) The region in which to obtain the CCE nodes. If omitted, the provider-level region will
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cce_node.md]: Specifies?

File: docs/data-sources/cce_node.md [hunk 3]
```diff
@@ -22,30 +22,32 @@ data "huaweicloud_cce_node" "node" {
...
+* `flavor_id` - The flavor ID to be used.
 
 * `availability_zone` - Available partitions where the node is located.
 
 * `os` - Operating System of the node.
 
 * `subnet_id` - The ID of the subnet which the NIC belongs to.
 
-* `ecs_group_id` - The ID of Ecs group which the node belongs to.
+* `ecs_group_id` - The ID of ECS group which the node belongs to.
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cce_node.md]: to which the node belongs

File: docs/resources/cce_namespace.md [hunk 4]
```diff
@@ -29,21 +29,21 @@ The following arguments are supported:
...
   Changing this will create a new namespace resource.
 
-* `name` - (Optional, String, ForceNew) Specifies the name of the namespace. Must be unique. This parameter can
-  contain a maximum of 63 characters, which may consist of lowercase letters, digits and hyphens (-), and must
-  start and end with lowercase letters and digits. Changing this will create a new namespace resource.
-  Exactly one of `name` or `prefix` must be provided.
+* `name` - (Optional, String, ForceNew) Specifies the unique name of the namespace.
+  This parameter can contain a maximum of 63 characters, which may consist of lowercase letters, digits and hyphens (-),
+  and must start and end with lowercase letters and digits. Changing this will create a new namespace resource.
+  This parameter and `prefix` are alternative.
```
Comment (Jason-Zhang9309) [inline docs/resources/cce_namespace.md]: The meaning of `alternative` and `exactly one of` are different, please reconsider this change.

File: docs/resources/cce_node_pool.md [hunk 5]
```diff
@@ -43,104 +43,105 @@ resource "huaweicloud_cce_node_pool" "node_pool" {
...
 
+* `extend_param` - (Optional, Map, ForceNew) Specifies the extended parameter.
+  Changing this parameter will create a new resource.
+  The available keys are as follows:
   + `agency_name` - Specifies the agency name to provide temporary credentials for CCE node to access other cloud
     services.
-  + `alpha.cce/NodeImageID` - This parameter is required when a custom image is used to create a BMS node.
-  + `dockerBaseSize` - The available disk space of a single docker container on the node in device mapper mode.
-  + `DockerLVMConfigOverride` - Docker data disk configurations. The following is an example default configuration:
+  + `alpha.cce/NodeImageID` - Specifies the custom image ID to which create the BMS nodes.
```
Comment (Jason-Zhang9309) [inline docs/resources/cce_node_pool.md]: to be used to create

File: docs/resources/cce_namespace.md (hunk 4 above)
Comment (Lance52259) [reply docs/resources/cce_namespace.md]: OFC, this modification is wrong.

File: docs/data-sources/cce_node.md (hunk 2 above)
Comment (Lance52259) [reply docs/data-sources/cce_node.md]: sure

File: docs/data-sources/cce_node.md (hunk 3 above)
Comment (Lance52259) [reply docs/data-sources/cce_node.md]: corrected

File: docs/resources/cce_node_pool.md (hunk 5 above)
Comment (Lance52259) [reply docs/resources/cce_node_pool.md]: corrected

File: docs/data-sources/cce_cluster.md (hunk 1 above)
Comment (Lance52259) [reply docs/data-sources/cce_cluster.md]: corrected

**Pull request merged**